apier/v1/resourcesv1.go

```diff
@@ -81,6 +81,9 @@ func (apierV1 *ApierV1) SetResourceProfile(res *engine.ResourceProfile, reply *s
 	if err := apierV1.DataManager.SetResourceProfile(res, true); err != nil {
 		return utils.APIErrorHandler(err)
 	}
+	if err := apierV1.DataManager.SetResource(&engine.Resource{Tenant: res.Tenant, ID: res.ID, Usages: make(map[string]*engine.ResourceUsage)}); err != nil {
```

Comment:
Please add a check, making sure we create new resource if none existing already since Set can update an existing and we will destroy previus